CI/travis.linux.after_success.sh

```diff
@@ -139,6 +139,7 @@ if { [ "${DEPLOY}" = "deploy" ]; } ||
 
       cd "${TRAVIS_BUILD_DIR}"
       # generate and upload the tarball
+      chmod +x "${HOME}/git-archive-all.sh"
```

Comment:
Is there any reason we should not just download and include this script directly in the repository (where we can set the mod directly) before adding it to the repository?

Reply:
It's not really a part of Mudlet, just needed for deployment.